spec/text-editor-component-spec.js

```diff
@@ -173,21 +173,19 @@ describe('TextEditorComponent', () => {
         expect(actualWidth).toBe(expectedWidth + 'px')
       }
 
-      {
-        // Make sure we do not throw an error if a synchronous update is
-        // triggered before measuring the longest line from a
-        // previously-scheduled update.
-        editor.getBuffer().insert(Point(12, Infinity), 'x'.repeat(100))
-        expect(editor.getLongestScreenRow()).toBe(12)
-
-        TextEditorComponent.getScheduler().readDocument(() => {
-          // This will happen before the measurement phase of the update
-          // triggered above.
-          component.pixelPositionForScreenPosition(Point(11, Infinity))
-        })
+      // Make sure we do not throw an error if a synchronous update is
+      // triggered before measuring the longest line from a
+      // previously-scheduled update.
+      editor.getBuffer().insert(Point(12, Infinity), 'x'.repeat(100))
+      expect(editor.getLongestScreenRow()).toBe(12)
 
-        await component.getNextUpdatePromise()
-      }
+      TextEditorComponent.getScheduler().readDocument(() => {
+        // This will happen before the measurement phase of the update
+        // triggered above.
+        component.pixelPositionForScreenPosition(Point(11, Infinity))
+      })
+
+      await component.getNextUpdatePromise()
```

Comment:
Does the linter mandate that we remove this block? 

This use of blocks is a pattern we employ to exercise multiple scenarios in a single test, without having to come up with unique variable names for each scenario.

Reply:
Yes, this is mandated by the [`no-lone-blocks`](https://eslint.org/docs/rules/no-lone-blocks) rule, it only complains when there are no variables defined inside the block.

If this is a common pattern across the codebase I can disable this rule for the `spec/` folders (I have to figure out how to do this since we're using the `standard` rules for linting).

Reply:
> If this is a common pattern across the codebase I can disable this rule for the `spec/` folders (I have to figure out how to do this since we're using the `standard` rules for linting).

That works for me.

Alternatively, could we use an `eslint-disable` comment to tell the linter to ignore that particular rule in this particular spec?